frontend/src/tests/pages/Earthquakes/EarthquakesCreatePage.test.js

```diff
@@ -0,0 +1,111 @@
+import { render, waitFor, fireEvent } from "@testing-library/react";
+import EarthquakesCreatePage from "main/pages/Earthquakes/EarthquakesCreatePage";
```

Comment:
The rest of these tests are failing for me, likely because we haven't gotten to their issues?